### DISCUSSION TITLE
Preguntas acerca del reporte TPAI Entrega 3

### DISCUSSION THREAD

**Discussion by julchat** [dds-jv/foro#149] [TPA - 2020] [answered]
Hola, tenemos unas dudas sobre como funciona el reporte que hay que hacer para las etiquetas.

Primero, ¿Cómo se supone que debe estar hecho?
Estaría bien hacer un System.Print de cada etiqueta, y los gastos de cada una? Un archivo? Como deberíamos mostrar eso? Se deja para la parte de presentación?

Segundo, cada entidad lo realiza cuando quiere (deberían hacerlo cada un mes) o debemos automatizarlo para que exactamente cada mes se efectue el reporte?

Gracias!

**Comment by gastonprieto** [comment, marked as answer]
Buenas!

La idea del reporte es que tengas un objeto de dominio que dado una etiqueta o etiquetas depende de como lo modeles y te devuelva esa información. Lo del `println` o generar un archivo eso es tema de presentación.

Cuando quiere, no hace falta automatizarlo.

**Comment by julchat** [comment]
Gracias!
